Source/Common/Values{T1,T2,T3,T4,T5,T6,T7}.cs

```diff
@@ -503,7 +505,9 @@ public Values(IEnumerable<object?> items)
         /// <param name="list">The list of values.</param>
         /// <returns>The result of the conversion.</returns>
 #pragma warning disable CA2225 // Operator overloads have named alternates
+#pragma warning disable CA1002 // Do not expose generic lists
```

Comment:
Is the warning wanting us to have `ICollection<T>` or `IList<T>` instead?

Also, similar to another comment, is it worth having this (and maybe the operator overload) pragma settings set for the whole file?